src/commands/core/plugins/add.ts

```
@@ -18,40 +19,43 @@ export default class CorePluginsAdd extends BaseCommand<typeof CorePluginsAdd> {
+            resolveSpinner.succeed(`Resolved as ${isCollection ? 'collection' : 'asset'}`)
+        } catch (error) {
+            resolveSpinner.fail(error instanceof Error ? error.message : 'Failed to resolve address')
+            throw error
         }
 
         if (flags.wizard) {
             const selectedPlugins = await pluginSelector({
-                filter: flags.collection ? PluginFilterType.Collection : PluginFilterType.Asset,
+                filter: isCollection ? PluginFilterType.Collection : PluginFilterType.Asset,
                 type: 'list',
                 managedBy: PluginFilterType.Authority
```

ESLint

[error] Expected "managedBy" to come before "type"

(perfectionist/sort-objects)